grammars/html.cson

```diff
@@ -587,12 +587,69 @@
         'include': '#unquoted-attribute'
       }
     ]
+  'tag-class-attribute':
+    'begin': '\\b(class)\\b\\s*(=)'
```

Comment:
This should be `\\b(class)(=)` instead, as spaces cannot follow the attribute name.

Reply:
Done.

Reply:
@50Wliu On what is "spaces cannot follow the attribute name" based? The HTML spec indicates that an attribute name is followed by "zero or more space characters".